application/constants/templates.py

```diff
@@ -43,3 +49,51 @@ class HookTypes(StrEnum):
     Template application hook types.
     """
     kubernetes_job = 'kubernetes_job'
+
+
+class http(StrEnum):
```

Comment:
PEP8. Class names should be in  Pascal case. It like Camel case but first later is capital.